task_list/app/assets/stylesheets/application.css

```diff
@@ -89,10 +89,6 @@
    text-align: left;
  }
 
- th, td {
-   padding-left: 50px;
- }
-
  a {
```

Comment:
For some of the styles below, you want to separate multi-word styles with the `-` rather than the `_`